org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DefaultVariableProvider.java

```diff
@@ -267,4 +273,33 @@ public Variable getThisVariable(StackFrame stackFrame) {
         });
         return res;
     }
+
+    public boolean isIncludeStatic() {
+        return includeStatic;
+    }
+
+    public void setIncludeStatic(boolean includeStatic) {
+        this.includeStatic = includeStatic;
+    }
+    
+    private void mergeDefaultOptions(IFormatter formatter) {
+        for (Map.Entry<String, Object> entry : formatter.getDefaultOptions().entrySet()) {
```

Comment:
Let's generate the default options each time the function is called without using cache to simply the problem first.
Adding cache should be a separate change list.